docs/developer/running/storage_operator.rst

```diff
@@ -0,0 +1,41 @@
+Running the storage operator locally
+====================================
+
+Requirements
+------------
+
+- `Go <https://golang.org/>`_ (1.12 or higher) and
+  `operator-sdk <https://github.com/operator-framework/operator-sdk>`_ (0.9 or
+  higher): to build the Kubernetes Operators
+- `Mercurial <https://www.mercurial-scm.org/>`_: some Go dependencies are
```

Comment:
For comprehensiveness: `git` is a dependency too ^^

Reply:
Yeah, but if you cloned the repo it's expected that you have `Git` ^^

If we go that way, a computer is required as well :stuck_out_tongue: